utilities/virt.py

```diff
@@ -2757,3 +2760,54 @@ def wait_for_virt_handler_pods_network_updated(
         )
         raise
     return False
+
+
+def create_vm_with_nginx_service(
+    namespace: Namespace,
+    client: DynamicClient,
+    utility_pods: list[Pod],
+    node: Node,
+    node_selector_label: Optional[dict] = None,
+):
+    name = "nginx"
+    with VirtualMachineForTests(
+        namespace=namespace.name,
+        name=name,
```

Comment:
should we strict to a hardcoded name? i.e what if someone wants more than one vm with nginx in the same ns?

Reply:
done